React-Template-master/src/components/Logo.tsx

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 import { Box, SxProps } from '@mui/material';
 import AppLogo from '../assets/Lejit.svg';
+import Correct from '../assets/Correct.svg';
+import Warning from '../assets/Warning.svg';
 // ----------------------------------------------------------------------
 
 interface LogoProps {
     sx?: SxProps;
 }
 
-export function Logo(sx: LogoProps): JSX.Element {
+export const Logo = (sx: LogoProps): JSX.Element => {
```

Comment:
Instead of replicating the same code again, can we add a prop to pass the icon to the logo component.

Reply:
Okay